### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN042.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#90] [General]
# https://blog.3place.cn/3place/lmyn/LMYN042.html

https://blog.3place.cn/3place/lmyn/LMYN042.html

**Comment by picklecai** [comment]
光辉的国王底离钵，
听到了叔祖们被杀，
愁绪满怀痛苦不堪，
却是想不出什么办法。 （1.41.5）

’怎样把恒河拉下天堂？
怎样为他们举行水祭？
怎样把他们度上天宫？’

要恒河下凡才能超度这6万个人？

**Comment by picklecai** [comment]
如果我的苦行获功果，
他们都应得到我的水祭，
萨竭罗所有儿子六万个。 （1.41.17）

那些高贵尊严的人们
骨灰如果得到恒河水冲，
那么，我们那一些祖先，
都将会永远地进入天宫。 （1.41.18）

终于看明白了。是“如果得到恒河水冲”。而现在还在原地，没有得到任何水。

**Comment by picklecai** [comment]
源于喜马拉雅山的恒河，
本是喜马拉雅山的长女。
想要扛得动她，国王呀！
你只有把湿婆去祷祈。 （1.41.22）

这个也有意思。
黄河之水天上来，奔流到海不复回。
飞流直下三千尺，疑是银河落九天。

李白，从来没想过是不是扛得动。
